### ISSUE TITLE
Too long all_eta

### ISSUE THREAD

**Issue by SuhoPark0706** [fawnliu/TRIS#3]
Thank you for your outstanding paper! I tried to retrain your model to use it as my baseline model.

![Screenshot from 2023-12-29 17-57-37](https://github.com/fawnliu/TRIS/assets/153984003/19b6f82c-114f-49fd-b4cb-2790c1cb2587)

This is my current state on training stage1.

I just checked that about 33seconds consumed on loss.backword().

Is it right that all_eta is logged as 5 days?

**Comment by SuhoPark0706** [comment]
conda install pytorch==1.13.1 torchvision==0.14.1 torchaudio==0.13.1 pytorch-cuda=11.6 -c pytorch -c nvidia

I solved it by changing torch version! thank you

**Comment by fawnliu** [comment]
Hi, @SuhoPark0706. I optimized my code, and it also works well on PyTorch 1.8.  Good luck!

<img width="813" alt="image" src="https://github.com/fawnliu/TRIS/assets/17973682/375cc41c-47a8-419d-91aa-24cfb391e5b6">
